docs/content/architecture/overview.md

```diff
@@ -115,40 +118,56 @@ Jobs and Control Edges make up a Workflow.
 
 Translator: It converts the AI Graph into the Workflow that the scheduler can execute.
 
-![Alt text](../images/flink-ai-flow/design/translator.jpg)
+![Alt text](../images/architecture/translator.png)
 
 The working steps of the translator are as follows:
 1. A group of AI Nodes with the same job config are combined into an AISubGraph.
 2. Call the corresponding JobGenerator according to the job config to convert all AISubGraphs to the corresponding Jobs.
 3. Add all Control Edges to the Workflow.
 
-## Meta Service
+## AIFlow Server
+AIFlow Server provides meta information management and scheduling services.
+
+### Meta Service
 
 Meta Service: Provides the CRUD(Create, Read, Update and Delete) service of metadata generated in the workflow of machine learning tasks.
 
 The services provided are shown in the figure below:
 
-![Alt text](../images/flink-ai-flow/design/meta.jpg)
+![Alt text](../images/architecture/meta.png)
 
 1. Dataset: It provides dataset meta-information services, including data address, data format and other information.
    Users can define the program for reading and writing data according to it.
 2. Project: It provides project meta-information services, 
    including project name, project description and other information.
    Users can organize the machine learning workflow of different projects based on it.
-3. Workflow&Jobs: It provides workflows and jobs meta-information services, 
-   including workflow description, job description and other information.
-   Users can view the submitted workflow's meta-information, 
-   and the job's meta-information according to it.
-4. Model Center: It provides Models meta-information services.
+3. Workflow: It provides workflows meta-information services.
+   Users can view the submitted workflow's meta-information according to it.
+4. Model: It provides Models meta-information services.
    It contains the description information of the model and the version information of the model.
-5. Metric: It provides metric meta-information services, such as model version evaluate metric, etc.
-6. Artifacts: It provides artifacts meta-information services such as configuration files, jar packages etc.
+5. Metric: It provides metric meta-information services, such as model version evaluate metric.
+6. Artifact: It provides artifacts meta-information services such as configuration files, jar packages etc.
+
+### Scheduling Service
+
+Scheduling Service: The Scheduling Service converts the workflow into an object that can be recognized by the scheduler 
+and submits it to the scheduler.
+The traditional scheduler can only schedule batch jobs (when upstream job finished then downstream jobs can run) 
+but in the online learning scenario where we have jobs that will never finish, it does not meet the demand. 
+The scheduler for Scheduler Service docking must support event-based scheduling.
```

Comment:
Add the introduction of the event-based scheduling. The users don't understand what the event-based scheduling is.